README.md

```diff
@@ -5,7 +5,7 @@
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-zoom)](https://github.com/mattermost/mattermost-plugin-zoom/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-zoom/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-zoom/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-**Maintainer:** [@mickmister](https://github.com/mickmister)
+**Maintainer:** [@wiggin77](https://github.com/wiggin77)
```

Comment:
Let's just remove the maintainers list for each plugin.

Reply:
@wiggin77 Done